src/operators.jl

```diff
@@ -370,6 +370,26 @@ function bind!(dest::Signal, src::Signal, twoway=true; initial=true)
 end
 
 """
+    `bind!(dest, src2dest, src, dest2src=nothing; initial=true)`
+
+for every update to `src` also update `dest` with a modified value (using the
+function `src2dest`) and, if `dest2src` is supplied, a two-way update will hold.
+If `initial` is false, `dest` will only be updated to `src`'s modified value
+when `src` next updates, otherwise (if `initial` is true) both `dest` and `src`
+will take the respective's modiefied values immediately. Note that currently 
+this can not be `unbind!`. Also note that for a two-way bind, a false initial
+is impossible.
```

Comment:
> Also note that for a two-way bind, a false initial is impossible.

Is that true? Doesn't look like it from the code, but maybe I'm missing something?

Reply:
There's a bug in twoway bind with `initial=false`, [this line](https://github.com/JuliaGizmos/Reactive.jl/blob/master/src/operators.jl#L375) should be `bind!(src, dest, false; initial = initial)`, anyway, I'll add it after I merge this, and remove this line from the docstring.